src/sse/sse_common.rs

```diff
@@ -90,6 +90,14 @@ macro_rules! boilerplate_fft_sse_oop {
                     fft_error_outofplace(self.len(), input.len(), output.len(), 0, 0);
                 }
             }
+            fn process_outofplace_with_scratch(
+                &self,
+                input: &mut [Complex<T>],
+                output: &mut [Complex<T>],
+                scratch: &mut [Complex<T>],
+            ) {
+                self.process_immutable_with_scratch(input, output, scratch);
```

Comment:
Unfortunately can't forward inside the boilerplante macros because of the error message thing